tools/packaging/kata-deploy/README.md

```diff
@@ -254,7 +270,7 @@ applying labels to the nodes.
 This DaemonSet installs the necessary Kata binaries, configuration files, and virtual machine artifacts on
 the node. Once installed, the DaemonSet adds a node label `katacontainers.io/kata-runtime=true` and reconfigures
 either CRI-O or containerd to register three `runtimeClasses`: `kata-clh` (for Cloud Hypervisor isolation), `kata-qemu` (for QEMU isolation),
-and `kata-fc` (for Firecracker isolation). As a final step the DaemonSet restarts either CRI-O or containerd. Upon deletion,
+, `kata-fc` (for Firecracker isolation) and `kata-stratovirt` (for StratoVirt isolation). As a final step the DaemonSet restarts either CRI-O or containerd. Upon deletion,
```

Comment:
nits: previous line already ends in a comma, Add a new line break to keep doc lines to the recommended 120 chars

```suggestion
`kata-fc` (for Firecracker isolation) and `kata-stratovirt` (for StratoVirt isolation).
As a final step the DaemonSet restarts either CRI-O or containerd. Upon deletion,
```

Reply:
thanks，updated.